code/modules/projectiles/guns/ballistic/launchers.dm

```diff
@@ -109,7 +109,25 @@
 		sleep(20)
 		return OXYLOSS
 
-
+/obj/item/gun/ballistic/foamfinger
+	name = "foamfinger"
```

Comment:
Make sure the admin one has the same `name` as the toy.

Reply:
done